install.ps1

```diff
@@ -49,6 +49,31 @@ foreach ($line in $resp.Content.Split("`n")) {
 Write-Host "Downloading $($releaseInfo.url)"
 Invoke-WebRequest -Uri $releaseInfo.url -OutFile 'buildkite-agent.zip'
 
+if (Test-Path -Path $installDir) {
+    $permissions = (Get-Acl $installDir).Access |
+        where {$_.IdentityReference -match 'BUILTIN\\Users' -and `
+            $_.AccessControlType -eq [System.Security.AccessControl.AccessControlType]::Allow}
+    if ($permissions) {
+        Write-Host "
+WARNING: All users have the following access to the installation directory '$installDir':
+WARNING:   $($permissions.FileSystemRights)
+WARNING: Consider only allowing administrators access to this directory.
+        "
+    }
+} else {
+    Write-Host "Restricting installation directory access to administrators"
```

Comment:
Is there enough context around this message output so that it's obvious what directory it's talking about?
Or should the message include `$installDir`?

Is it my imagination or does Windows generally capitalise **A**dministrators?

```suggestion
    Write-Host "Restricting installation directory access to Administrators: '$installDir'"
```

Reply:
Good point, I like it. Change applied.